app/src/main/java/tech/relaycorp/letro/account/model/Account.kt

```diff
@@ -27,6 +27,7 @@ data class Account(
     val veraidPrivateKey: ByteArray,
     val veraidMemberBundle: ByteArray? = null,
     val awalaEndpoint: String? = null,
+    val publicThirdPartyNodeId: String? = null,
```

Comment:
This name is misleading because the account doesn't have a public third-party endpoint. I'd suggest this:

```suggestion
    val veraidAuthEndpointId: String? = null,
```

Reply:
Done